core/iwasm/libraries/lib-pthread/lib_pthread_wrapper.c

```diff
@@ -648,6 +685,8 @@ pthread_create_wrapper(wasm_exec_env_t exec_env,
         wasm_runtime_free(info_node);
     if (routine_args)
         wasm_runtime_free(routine_args);
+    if (new_c_api_func_imports)
+        wasm_runtime_free(new_c_api_func_imports);
```

Comment:
Should not free again, one is the allocated memory is `*new_c_api_func_imports`, but not `new_c_api_func_imports`, see L634, the other is that the `*new_c_api_func_imports` is `new_module_inst->e->c_api_func_imports`, if it was set, it will have been freed in L683 with `wasm_deinstantiate` and `aot_deinstantiate`, see:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/interpreter/wasm_runtime.c#L2167-L2168